be/src/vec/columns/column_dictionary.h

```diff
@@ -297,6 +307,23 @@ class ColumnDictionary final : public COWHelper<IColumn, ColumnDictionary<T>> {
             return -1;
         }
 
+        inline StringValue& get_value(T code) { return _dict_data[code]; }
+        
+        inline void generate_hash_values() {
+            if (_hash_values.size() == 0) {
+                _hash_values.reserve(_dict_data.size());
```

Comment:
why not `resize` and `_hash_values[i] = HashUtil::murmur_hash3_32(sv.ptr, sv.len, 0);`

Reply:
It looks no difference.
Is there any benefit to your approach?

Reply:
1. less code
2. more quickly. each time you push back need to do `_size ++`

Reply:
👌🏻